lib/jazzy/config.rb

```diff
@@ -226,6 +226,17 @@ def expand_path(path)
                    'comments.',
       default: false
 
+    config_attr :lint_report,
+      description: ['Sets the path to which a lint report will be written. If',
+                    'no path is provided, no lint report will be written.',
```

Comment:
writing this file is fairly cheap compared to the rest of the disk IO jazzy does, so why not just default to `undocumented.json` where the current `undocumented.txt` would be?